docs/production-deployment/cloud/metrics/reference.mdx

```diff
@@ -26,146 +26,282 @@ keywords:
-| `resource_exhausted_cause` | Cause for resource exhaustion.                                                                                                                                                                                                                                            |
-| `task_type`                | Activity or Workflow.                                                                                                                                                                                                                                                     |
-| `temporal_account`         | Temporal Account.                                                                                                                                                                                                                                                         |
-| `temporal_namespace`       | Temporal Namespace.                                                                                                                                                                                                                                                       |
-| `temporal_service_type`    | Frontend or Matching or History or Worker.                                                                                                                                                                                                                                |
-| `is_background`            | This label on `temporal_cloud_v0_total_action_count` indicates when actions are produced by a Temporal background job, for example: hourly Workflow Export.                                                                                                               |
-| `namespace_mode`           | This label on `temporal_cloud_v0_total_action_count` indicates if actions are produced by an active vs a passive Namespace. For a regular Namespace, `namespace_mode` will always be “active”.                                                                            |
-
-The following is an example of how you can filter metrics using labels:
-
-```text
-temporal_cloud_v0_poll_success_count{__rollup__="true", operation="TaskQueueMgr", task_type="Activity", temporal_account="12345", temporal_namespace="your_namespace.12345", temporal_service_type="matching"}
-```
+* [How to set up Grafana with Temporal Cloud observability](/cloud/metrics/prometheus-grafana).
+* [How to monitor Worker Health with Temporal Cloud Metrics](/production-deployment/cloud/worker-health)
+* [How to monitor Service Health with Temporal Cloud Metrics](/production-deployment/cloud/service-health)
```

Comment:
You can use the `RelatedReadComponent` here if you want -

Example usage:

```
import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
<RelatedReadContainer>
  <RelatedReadItem path="/cloud/metrics/prometheus-grafana" text="How to set up Grafana with Temporal Cloud observability" archetype="feature-guide" />
  <RelatedReadItem path="/production-deployment/cloud/worker-health" text="How to monitor Worker Health with Temporal Cloud Metrics" archetype="feature-guide" />
  <RelatedReadItem path="/production-deployment/cloud/service-health" text="How to monitor Service Health with Temporal Cloud Metrics" archetype="feature-guide" />
</RelatedReadContainer>
```

Reply:
This is great. Doing that!

Reply:
[ERROR] Client bundle compiled with errors therefore further build is impossible.
Error: MDX compilation failed for file "/Users/ericasadun/Desktop/Sites/sites/documentation/docs/production-deployment/cloud/metrics/reference.mdx"
Cause: Unexpected `ExpressionStatement` in code: only import/exports are supported
Details:
{
  "column": 1,
  "message": "Unexpected `ExpressionStatement` in code: only import/exports are supported",
  "line": 89,
  "name": "89:1-93:24",
  "place": {
    "start": {
      "line": 89,
      "column": 1,
      "offset": 3660
    },
    "end": {
      "line": 93,
      "column": 24,
      "offset": 4184
    }
  },
  "reason": "Unexpected `ExpressionStatement` in code: only import/exports are supported",
  "ruleId": "non-esm",
  "source": "micromark-extension-mdxjs-esm",
  "url": "https://github.com/micromark/micromark-extension-mdxjs-esm#unexpected-type-in-code-only-importexports-are-supported"
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this comm

Reply:
Moving the import to just after the preface allows it to build. Looks awesome
![snapshot-06-11-24@13-29-15](https://github.com/temporalio/documentation/assets/156466156/1797aef8-8f3a-43e0-8446-7cccc11d07c6)